### ISSUE TITLE
Edit Q&A report workflow filter

### ISSUE THREAD

**Issue by chri5tia** [department-of-veterans-affairs/va.gov-cms#15891]
## Background
Work was performed by the Public Websites team in order to fix the issue wherein inaccurate Menu link breadcrumbs on the "Board of Veterans Appeals" page on prod > the “Manage a legacy VA appeal” link is causing hourly broken link alerts, which could possibly mask actual errors that need to be addressed due to the 'noise' of a faux error.

It was determined the fix coded should use a different editorial workflow filter, and this ticket was created in order to address that.

## User Story or Problem Statement
[Recently created and merged Q&A content](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/15833) view needs to be updated to use a different editorial workflow filter, per style guide.

https://prod.cms.va.gov/admin/structure/cm_document/note/124/views-style-guide

See this slack convo for context:
https://dsva.slack.com/archives/C52CL1PKQ/p1698437144686279

There is some urgency in prioritizing this as it will cause issues in the site when other work is merged e.g. it's holding up [other PRs](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/15872).

## Acceptance Criteria
- [x] View uses the Multiple Workflow filter
- [ ] Feedback from first PR is addressed: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/15896
- [ ] Request CAIA feedback in #sitewide-content-accessibility-ia channel using prod CMS link

**Comment by FranECross** [comment]
@chri5tia Approved to be worked on right away so as not to hold up other PRs or cause potential prod issues. Thank so much! cc @jilladams @swirtSJW

**Comment by jilladams** [comment]
Outstanding PR feedback that we opted not to address in #15896 in order to unblock Facilities, but which does need to be addressed before this ticket closes. I'm noting what I see in PR and can't tell if it was already addressed.  @chri5tia could you check boxes for anything that did already get handled?

- [ ] Page specific Q&A view should move to /admin/content/audit/page-specific-qa and display a tab in alpha order 
  - [ ] Local task only: checked
  - [ ] Local task link title: "Page-specific Q&A"
  - [ ] Local task parent: "content(content_audit_page)"
- [ ] Content type column should be sortable

**Comment by chri5tia** [comment]
@jilladams all addressed in follow-up PR
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/16003

**Comment by chri5tia** [comment]
Checked on prod and https://prod.cms.va.gov/admin/content/audit/qa doesn't load. I need to see what happened.

**Comment by jilladams** [comment]
@dsasser noting since we're transitioning Christia's work to you: 

She made a code change [here](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/16003/files#diff-276a4737b1e445a09fe15d50076814871b0d62c7ab8d092b3905a9ae4a8f40d6R7992) to move this view from `/admin/content/qa` to `/admin/content/audit/qa`, but that still isn't true in prod. This is the last dangler she had from last sprint, so we'll need your help to sort out why.

**Comment by jilladams** [comment]
This view did show up in the right path, so ticket is closed. I did notice, though, that the tab is not in alpha order. ( Poked around a little in https://github.com/department-of-veterans-affairs/va.gov-cms/blob/8c99f59a3bb1245e407385a177d619a71ee301d2/config/sync/views.view.content.yml to see if I could figure out how to rearrange it, but: no. Could not.)  Is that easy to toss a fix up, or should I ticket?

**Comment by chri5tia** [comment]
It is because the weight of the menu tab option is set to -150 and that likely needs to be erased. Commenting because I checked my email and realized I didn't put that in here yesterday.

**Comment by dsasser** [comment]
@jilladams 

> This view did show up in the right path, so ticket is closed. I did notice, though, that the tab is not in alpha order. ( Poked around a little in https://github.com/department-of-veterans-affairs/va.gov-cms/blob/8c99f59a3bb1245e407385a177d619a71ee301d2/config/sync/views.view.content.yml to see if I could figure out how to rearrange it, but: no. Could not.) Is that easy to toss a fix up, or should I ticket?



PR is up here: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/16085

**Comment by jilladams** [comment]
Verified in prod.
